tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrNotAllowedNode.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: %dxc -T lib_6_7 %s | FileCheck %s
```

Comment:
This should be lib_6_8. I suspect once you get past the initial error, you'll get one about that (or should anyway)